octobot_tentacles_manager/configuration/tentacles_setup_configuration.py

```diff
@@ -197,14 +197,12 @@ def _set_activation_using_default_config(self, tentacle, tentacle_class_name, de
             if tentacle_class_name in default_config[tentacle.tentacle_root_type]:
                 self.tentacles_activation[tentacle.tentacle_root_type][tentacle_class_name] = \
                     default_config[tentacle.tentacle_root_type][tentacle_class_name]
-            else:
-                # activate by default unless the sub type of this tentacle in among the sub types to be deactivated by
-                # default
-                self.tentacles_activation[tentacle.tentacle_root_type][tentacle_class_name] = \
-                    tentacle.get_simple_tentacle_type() not in self.DEFAULT_DEACTIVATABLE_TENTACLE_SUB_TYPES
-        else:
-            # if tentacle_type not in default config: activate by default
-            self.tentacles_activation[tentacle.tentacle_root_type][tentacle_class_name] = True
+                return
+
+        # activate by default unless the sub type of this tentacle in among the sub types to be deactivated by default
+        # or if tentacle_type not in default config: activate by default
+        self.tentacles_activation[tentacle.tentacle_root_type][tentacle_class_name] = \
+            tentacle.get_simple_tentacle_type() not in self.DEFAULT_DEACTIVATABLE_TENTACLE_SUB_TYPES
```

Comment:
this block is not in `else` ?

Reply:
No it's should be the `else` of the two `if` so I just added a `return`

Reply:
Oh, right !